elastic/kibana
# Add helper text in the edit rule form about the change in privileges when saving the rule

Fixes #131238

In this PR, I'm adding an icon + tooltip as described in https://github.com/elastic/kibana/issues/131238#issuecomment-1119491760 to warn users about the change in privileges.

Note: Code in `rule_edit.tsx` might be easier to review if you view it with whitespace ignored.

Updated UI:
<img width="596" alt="Screen Shot 2022-05-06 at 1 23 56 PM" src="https://user-images.githubusercontent.com/3694571/167181975-b19c64c3-5901-4acf-aee9-32bc28abccd2.png">

## To verify - scenario 1
1. Create an alerting rule
2. Open the edit flyout of the newly created rule
3. Notice the new icon + toolip in the flyout footer

## To verify - scenario 2
1. Start Elasticsearch with `-E xpack.security.enabled: false` and Kibana as usual
2. Create an alerting rule
3. Open the edit flyout of the newly created rule
4. Notice the absence of the icon + toolip in the flyout footer

cc @mdefazio
